engine/src/main/java/com/datatorrent/stram/security/StramWSFilter.java

```diff
@@ -99,6 +99,7 @@ public void init(FilterConfig conf) throws ServletException
         proxyAddresses = new HashSet<String>();
         for (String proxyHost : proxyHosts) {
           try {
+            logger.debug("resolving proxy hostname {}", proxyHost);
```

Comment:
Not every request but periodically (5mins) and its in debug so normally it won't be logged unless asked for.